src/tests/JIT/HardwareIntrinsics/Arm/Shared/LoadAndInsertScalarTest.template

```diff
@@ -161,6 +165,7 @@ namespace JIT.HardwareIntrinsics.Arm
             for (var i = 0; i < Op1ElementCount; i++) { _data1[i] = {NextValueOp1}; }
             Unsafe.CopyBlockUnaligned(ref Unsafe.As<{Op1VectorType}<{Op1BaseType}>, byte>(ref _fld1), ref Unsafe.As<{Op1BaseType}, byte>(ref _data1[0]), (uint)Unsafe.SizeOf<{Op1VectorType}<{Op1BaseType}>>());
 
+            _fld2 = {ElementIndex};
```

Comment:
Lot of the tests that takes immediate value is missing this coverage. We should fix it some day.

Reply:
Added an issue: https://github.com/dotnet/runtime/issues/108060